deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -140,6 +140,13 @@ spec:
                     type: object
                     description: Spec to control the desired behavior of storage cluster rolling update.
                     properties:
+                      minReadySeconds:
```

Comment:
do we make this term or there's a similar term on k8s side?

Reply:
there is minReadySeconds in k8s deployment spec.